python/guide/packages/gluon/loss.md

```diff
@@ -121,4 +166,120 @@ y = nd.array([0, 1])
 loss(x, y)
 ```
 
-For binary classification, we can use `SigmoidBinaryCrossEntropyLoss`.
+#### [Hinge Loss](https://mxnet.incubator.apache.org/api/python/gluon/loss.html#mxnet.gluon.loss.HingeLoss)
+
+Classifcation problems normally require a zero-one loss function, which assigns 0 loss to correct classifications and 1 otherwise. The problem of such a function is that it is hard to optimize and its gradients would be zero. Hinge Loss creates an upper bound on the zero-one loss function which makes it a convex and continous function.  It is defined as:
```

Comment:
"Classifcation problems normally require a zero-one loss function, which assigns 0 loss to correct classifications and 1 otherwise." this isn't the case with cross entropy though?

Reply:
Can you give an example of when this is used.

Reply:
done.